moa/src/main/java/moa/classifiers/meta/PairedLearners.java

```diff
@@ -0,0 +1,126 @@
+package moa.classifiers.meta;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+import moa.classifiers.AbstractClassifier;
+import moa.classifiers.Classifier;
+import moa.core.Measurement;
+import moa.options.ClassOption;
+import moa.options.FloatOption;
+import moa.options.IntOption;
+import weka.core.Instance;
+import weka.core.Utils;
```

Comment:
Please use the new instance (com.yahoo.labs.samoa.instances.Instance), and import moa.core.MiscUtils instead of weka.core.Utils.